src/sales/order-workflow.md

```diff
@@ -8,13 +8,13 @@ Step 1: Place Order
 : The checkout process begins when the shopper clicks the Go to Checkout button on the shopping cart page or [reorders]({% link sales/reorders-allow.md %}) directly from their customer account.
 
 Step 2: Order Pending
-: In the Orders grid, the status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled.
+: The status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.
```

Comment:
```suggestion
: The initial sales order status is `Pending`. In this state, the payment has not been processed and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.
```